src/components/WalletModal/index.tsx

```diff
@@ -152,6 +187,27 @@ export default function WalletModal({
   const walletModalOpen = useModalIsOpen(ApplicationModal.WALLET)
   const toggleWalletModal = useToggleWalletModal()
 
+  const allTokens = useAllTokens()
+  const [tokenBalances, tokenBalancesIsLoading] = useAllTokenBalances()
+  const sortedTokens: Token[] = useMemo(() => {
+    void tokenBalancesIsLoading // creates a new array once balances load to update hooks
```

Comment:
not sure what this void is doing?

Reply:
lifted logic from CurrencySearch.tsx, I think the original goal was just to have tokenBalancesIsLoading as a dependency in the useMemo hence the void, but I made the logic more explicit now with the ternary. also udpated CurrencySearch.tsx.